### ISSUE TITLE
While loop is executing one time after it should

### ISSUE THREAD

**Issue by IAmCheeseman** [BergerAPI/ACL#2]
Code:
```js
let true = 1
let false = 0

let inWhile = true

let i = 1
while inWhile {
    print("While loop is at " + i)
    if i == 5 {
        inWhile = false
        print("Exiting while loop!")
    }
    i = i + 1
}
```
Output:
```
While loop is at 1
While loop is at 2
While loop is at 3
While loop is at 4
While loop is at 5
Exiting while loop!
Exiting while loop!
```

**Comment by BergerAPI** [comment]
Fixed in commit 575cfd2

**Comment by BergerAPI** [comment]
<img width="162" alt="Screen Shot 2021-12-31 at 12 56 52 PM" src="https://user-images.githubusercontent.com/58854363/147822126-8e72981d-750f-473a-88eb-1c689fe1136e.png">
